app/models/profile_field.rb

```diff
@@ -1,9 +1,7 @@
 class ProfileField < ApplicationRecord
-  self.ignored_columns = ["group"]
-
-  before_create :generate_attribute_name
+  include ActsAsProfileField
 
-  WORD_REGEX = /\w+/.freeze
+  self.ignored_columns = ["group"]
```

Comment:
This will eventually get removed, but I left it in for now as it has nothing to do with this PR.